src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/VirtualSecureModeEnclaveProvider.cs

```diff
@@ -274,32 +272,25 @@ public EnclaveReportPackageHeader(byte[] payload)
             offset += sizeof(uint);
         }
 
-        public int GetSizeInPayload()
-        {
-            return 6 * sizeof(uint);
-        }
+        public static int SizeInPayload => 6 * sizeof(uint);
     }
 
     // A managed model of struct VBS_ENCLAVE_REPORT
     // https://msdn.microsoft.com/en-us/library/windows/desktop/mt844255(v=vs.85).aspx
     internal class EnclaveReport
     {
-        private int Size { get; set; }
+        private const int EnclaveDataLength = 64;
```

Comment:
I can see that `Size` is only set once in the ctor and this is private, so not sure why did we needed it in the first place so removing it seem fine.